runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -406,6 +414,122 @@ chpl_comm_taskPrvData_t* get_comm_taskPrvdata(void) {
 }
 
 
+////////////////////////////////////////
+//
+// task local buffering
+//
+
+// Largest size to use unordered transactions for
+#define MAX_UNORDERED_TRANS_SZ 1024
+
+//
+// Maximum number of PUTs/AMOs in a chained transaction list.  This
+// is a provisional value, not yet tuned.
+//
+#define MAX_TXNS_IN_FLIGHT 64
+
+#define MAX_CHAINED_PUT_LEN MAX_TXNS_IN_FLIGHT
```

Comment:
CHAINED seems like the wrong word (since it's a ugni terminology) I'm fine with keeping it, but seems a little off.